### ISSUE TITLE
reopen minimized app with sidebar using crosswalk.

### ISSUE THREAD

**Issue by blindsz** [shprink/ionic-native-transitions#122]
When I minimize my app and reopen the app the sidebar flashes for a second. It's like a blinking sidebar is appearing when your resume the app. Please I really need anyone help. Thank you!

**Comment by thgfrzm** [comment]
It's cached?

**Comment by blindsz** [comment]
Yes, I think so.

**Comment by blindsz** [comment]
Not just that. It also flashes the last view you entered. for example you are from view A and transfer to view B. when you minimize your app that in the state of view B when you resume the app view A flashes.

**Comment by blindsz** [comment]
I don't think that there something to do with the cache or with angular or ionic I think this is the problem when you set to true  -->  <preference name="CrosswalkAnimatable" value="true" />

**Comment by blindsz** [comment]
Anyone there can help?
